index.html

```diff
@@ -91,6 +95,9 @@ <h1>Boot up your dreams.
 				</h1>
 				<div class="btn-container">
 					<a data-link="about"><button class="splashbtn">Learn more about us</button></a>
+					<a><button class="splashbtn" onClick="window.location.href='https://ugeorgia.ca1.qualtrics.com/jfe/form/SV_82Gn6acN70D0wg6'">
```

Comment:
See above comment about combining the button link with the banner since the banner is always going to be visible so having them together may make more sense